toplevel/toploop.mli

```diff
@@ -195,3 +195,11 @@ val override_sys_argv : string array -> unit
    This is called by [run_script] so that [Sys.argv] represents
    "script.ml args..." instead of the full command line:
    "ocamlrun unix.cma ... script.ml args...". *)
+
+val split_path : string -> string list
+(** [split_path path] splits [path] according to the PATH-splitting conventions
+    of the platform. On Unix, this is exactly [String.split_on_char ':' path].
+    On Windows, entries are separated by semicolons. Sections of entries may be
+    double-quoted (which allows semicolons in filenames to be quoted). The
+    double-quote characters are stripped (i.e. [f"o"o = foo]; also
+    [split_path "foo\";\";bar" = ["foo;"; "bar"]) *)
```

Comment:
For my own curiosity, where is this quoting behaviour documented?

Reply:
It turns out I'd forgotten where I learned it from, so having had to search both the Internet and my memory, I added a comment! Amusingly, the comment in getpath.cpp is very clearly from OS/2 (i.e. 1988 or 1989)! 😂